dev/a11y_assessments/lib/use_cases/headings.dart

```diff
@@ -0,0 +1,52 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+import 'use_cases.dart';
+
+class HeadingsUseCase extends UseCase {
```

Comment:
This may not be a good use case for GAR, If we want to show case this we should probably provide a more comprehensive widget that uses heading semantics. for example Appbar or expansionTile

Reply:
If you have an article with plain text headings and paragraphs, what would be the idiomatic way to express it in Flutter? For example, let's take this wikipedia page - https://en.wikipedia.org/wiki/Hawaii. It has headings, such as "Etymology" and "Spelling of state name" at different levels. What widgets would you use to build this page in Flutter, and how communicate heading levels to the semantics correctly?

Reply:
I think that will be our customer's responsibility to verify GAR if they compose our widgets to create a page. Of course we need make sure provide enough API for them to do so, but I think that should be a unit test instead.

The scope of a11y_assessment is to test individual material or cupertino components that we provide, so I don't think this use case fit in the assessment.